spawns.lua

```diff
@@ -128,38 +128,40 @@ NorthGeorgiaFARPTransportSpawns = {
 
 -- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
 
-BladderPos = {}
-BladderPos.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
-BladderPos.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+WeatherPositioning = {}
+WeatherPositioning.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
+WeatherPositioning.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+WeatherPositioning.hClearance		= 305		-- meters [1000']: Default clearance to cloud
 
-function BladderPos.InitOnGroup(planeGroup, vSpeed)
-	local planeGroupName	= planeGroup:getName()
-	vSpeed = vSpeed or BladderPos.vSpeed
-	
+function WeatherPositioning.getCloudFreeAltitude()
 	local clouds = env.mission.weather.clouds
 	
-	-- Calculate orbit height. Ignore for partial cloud conditions
-	local hOrbit = BladderPos.hMaxFlightAlt
-	if (((clouds.base + clouds.thickness) >= hOrbit - 305) and (clouds.density >= 7)) then
-		hOrbit = math.min(clouds.base - 305, BladderPos.hMaxFlightAlt)
+	local hAltitude = WeatherPositioning.hMaxFlightAlt
+	if (((clouds.base + clouds.thickness) >= hAltitude - WeatherPositioning.hClearance) and (clouds.density >= 7)) then
+		hAltitude = math.min(clouds.base - WeatherPositioning.hClearance, WeatherPositioning.hMaxFlightAlt)
 	end
+	return hAltitude
+end
+
+function WeatherPositioning.avoidCloudLayer(planeGroup, vSpeed)
```

Comment:
Can we put a doc here to state that `vSpeed` is in meters per second instead of knots?